code/controllers/subsystem/air.dm

```diff
@@ -272,12 +272,61 @@ var/datum/subsystem/air/SSair
 			continue
 		T.Initalize_Atmos(times_fired)
 
-
 	if(active_turfs.len)
-		warning("There are [active_turfs.len] active turfs at roundstart, this is a mapping error caused by a difference of the air between the adjacent turfs. You can see its coordinates using \"Mapping -> Show roundstart AT list\" verb (debug verbs required)")
+		var/starting_ats = active_turfs.len
+		sleep(world.tick_lag)
+		var/timer = world.timeofday
+		warning("There are [starting_ats] active turfs at roundstart, this is a mapping error caused by a difference of the air between the adjacent turfs. You can see its coordinates using \"Mapping -> Show roundstart AT list\" verb (debug verbs required)")
 		for(var/turf/T in active_turfs)
 			active_turfs_startlist += text("[T.x], [T.y], [T.z]\n")
 
+		//now lets clear out these active turfs
+		var/list/turfs_to_check = active_turfs.Copy()
+		do
+			var/list/new_turfs_to_check = list()
+			for(var/turf/open/T in turfs_to_check)
+				new_turfs_to_check += T.resolve_active_graph()
+
+			active_turfs += new_turfs_to_check
+			turfs_to_check = new_turfs_to_check
+
+		while (turfs_to_check.len)
```

Comment:
Can you add a debug define to print the x and y + path of every affected turf? Might help me figure out what the mappers are doing wrong